xds/src/test/java/io/grpc/xds/XdsClientImplTest.java

```diff
@@ -2670,6 +2680,9 @@ private static Cluster buildCluster(String clusterName, @Nullable String edsServ
       clusterBuilder.setLrsServer(
           ConfigSource.newBuilder().setSelf(SelfConfigSource.getDefaultInstance()));
     }
+    if (upstreamTlsContext != null) {
```

Comment:
nit/optional: this `null` check unnecessary

Reply:
It is. `ClusterBuilder.setTlsContext` generates NPE